### DISCUSSION TITLE
Please add HuggingChat support.

### DISCUSSION THREAD

**Discussion by Abdalnablse10** [WAppAI/assistant#219] [Ideas]
Would be cool to have other options other than bing.

**Comment by Luisotee** [comment]
We are currently trying to add OpenRouter to be able to access every LLM supported by it.

**Reply by Abdalnablse10** [reply]
That would be nice, one question: Is it possible for local llms support with llama.cpp?

**Reply by Luisotee** [reply]
Probably but it would be a lot of work I think

**Reply by Abdalnablse10** [reply]
You can take a look at this if you want https://github.com/withcatai/node-llama-cpp
